zlib-rs/src/inflate.rs

```diff
@@ -914,7 +914,7 @@ impl<'a> State<'a> {
     }
 
     fn check(&mut self) -> ReturnCode {
-        if self.wrap != 0 {
+        if !cfg!(feature = "__internal-fuzz-disable-checksum") && self.wrap != 0 {
```

Comment:
I think this incidentally closes https://github.com/trifectatechfoundation/zlib-rs/issues/115, FYI @fintelia.